docs/tutorials/using-qos-ensurance.md

```diff
@@ -0,0 +1,165 @@
+# Qos Ensurance
+QoS ensurance guarantees the stability of the pods running on Kubernetes.
+Disable schedule, throttle, evict will be applied to low priority pods when detect the high priority pods be impacted by resource competition.
+
+# Qos Ensurance Architecture
+Qos ensurance's architecture as the below image shows. It contains three modules.
+1. metrics collector: collect metrics periodically
+2. anomaly analyzer: analyze the node triggered anomaly used collected metrics
+3. avoidance action: execute avoidance actions, include disable scheduling, throttle and eviction.
+
+Below is the architecture image.
+<div align="center"><img src="../images/crane-qos-ensurance.png" style="width:900px;" /></div>
+
+The main processes included the following steps:
+1. Metrics collector synchronized policies from kube-apiserver.
+2. Check to see if the policies have been changed. If the policies are changed, the metrics collector updates the collectors.
+3. Metrics collector collects metrics periodically.
+4. Transmit metrics to anomaly analyzer.
+5. Range all rules to analyze the avoidance threshold or the restored threshold reached.
+6. Merged for the analyzed results and notice the avoidance actions.
```

Comment:
6. Anomaly analyzer merges for the analyzed results and notice the avoidance actions.